mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_folder_notification_config_test.go

```diff
@@ -0,0 +1,125 @@
+package securitycenterv2_test
+
+import (
+	"testing"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccSecurityCenterV2FolderNotificationConfig_basic(t *testing.T) {
+	t.Parallel()
+
+	orgID := envvar.GetTestOrgFromEnv(t)
+	randomSuffix := acctest.RandString(t, 10)
+
+	context := map[string]interface{}{
+		"org_id":        orgID,
+		"random_suffix": randomSuffix,
+	}
+
+	configID := fmt.Sprintf("tf-test-config-%s", randomSuffix)
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"random": {},
+			"time":   {},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecurityCenterV2FolderNotificationConfig_basic(context),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_scc_v2_folder_notification_config.default", "config_id", configID),
+				),
+			},
+			{
+				Config:            testAccSecurityCenterV2FolderNotificationConfig_basic(context),
+				ResourceName:      "google_scc_v2_folder_notification_config.default",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"folder", "config_id",
+				},
```

Comment:
[We could add a post_import ](https://googlecloudplatform.github.io/magic-modules/develop/custom-code/#inject-code-before--after-crud-operations-and-import:~:text=post_import%3A%20templates/terraform/post_import/PRODUCT_RESOURCE.go.erb)to the FolderNotificationConfig.yaml file to set the values of folder and config_id from the import id during Import. Would you be happy to do this? This would allow those fields to not be ignored in these tests and would ensure that the resource block the resource is imported into has correct values for those fields